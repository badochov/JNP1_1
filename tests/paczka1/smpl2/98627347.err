Error in line 6: A89 S802 6,0
Error in line 12: DXyJGXf S802 32,1
Error in line 25: A89 A89 13,4
Error in line 33: xNUf08UTNE S715 74,8
