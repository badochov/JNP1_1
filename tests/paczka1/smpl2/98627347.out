A217 42,4
A89 A 264,1
BSAZ A 42,4
qa3psYFcDE S 86,6
xNUf08UTNE S 262,8
A89 264,1
A217 42,4
S715 262,8
S802 86,6
S715 439,3
A89 A 264,1 S 91,7
BSAZ A 42,4 S 31,0
DXbwtC A 98,2
DXyJGXf A 27,8 S 145,5
qa3psYFcDE S 86,6
wbWynpy8O5 A 21,4
xNUf08UTNE A 207,6 S 262,8
A89 362,3
A217 299,2
S715 439,3
S802 178,3
xNUf08UTNE A 207,6 S 262,8
